components/portal/main/partials/main-menu.html

```diff
@@ -50,7 +50,31 @@
         <span><md-icon>notifications</md-icon></span>
         <md-tooltip class="top-bar-tooltip" md-direction="bottom" md-delay="500">Notifications</md-tooltip>
       </md-button>
-      <username></username>
+      <!-- GUEST MODE ONLY -->
+      <md-button
+         ng-href="{{ MISC_URLS.myuwHome }}"
+         slot="myuw-profile"
+         ng-show="sessionCtrl.guestMode"
+         aria-label="Log in to {{ portal.theme.title }}"
+         target="_self"
+         class="md-default header__guest-login-btn">
+        Log In
+      </md-button>
+
+    <!-- NOT GUEST MODE -->
+    <myuw-profile
+      ng-hide="sessionCtrl.guestMode"
+      aria-label="Log in to {{ portal.theme.title }}"
```

Comment:
This `aria-label` doesn't look right. This is the not-guest-mode, right, where the user is already logged in?

Reply:
Yes! Thank you, I changed it!